### ISSUE TITLE
github action to track missing translation

### ISSUE THREAD

**Issue by rperon** [rperon/epic-stack-with-i18n#11]
Feedback from @fredericrous:

Nice to have final feature: a github action that checks there are no missing translation (some in the code but not in the translation json file)

This might be done by using i18n-parser, need some investigation to find the right solution.

**Comment by fredericrous** [comment]
yup, I was thinking of comparing the output of  i18n-parser with the version file.
a very basic implem could be:
- load the output with something like `Object.keys(result)`
- if there are more keys in the code than in the translation files
- loop over to find which are the missing keys with something like `output.filter(x => translation['en].includes(x));`
- do the opposite to know if there are keys in the translation files that are not in the code

Update: also the action should fail if there is a duplicated key in the same file!
